src/App.vue

```diff
@@ -2,32 +2,24 @@
   <div id="app">
     <button @click="getTotalSupply">Get Total MltiToken Supply</button>
     <form v-on:submit.prevent>
-      <input type="text" placeholder="Number of Tokens to Mint" v-model="tokenAmount" />
-      <button prevent="submit" @click="mintTokens">Mint Tokens</button>
+      <p>
+        <input type="text" placeholder="Number of Tokens to Mint" v-model="tokenAmount" />
+      </p>
+      <p>
+        <button prevent="submit" @click="mintTokens">Mint Tokens</button>
+      </p>
     </form>
     <hr />
     <pre v-if="response">{{ response | prettyJSON }}</pre>
   </div>
 </template>
 
 <script>
-import axios from 'axios';
-import axiosCookieJarSupport from '@3846masa/axios-cookiejar-support';
-import tough from 'tough-cookie';
+// [NEED TO FILL] The local address where the demo app will be served
+const BASE_URL = 'https://rjlrpdjikze7fn2vscxfuk365y.multibaas.com';
```

Comment:
See the note about API keys - it applies to specific URLs as well!

Reply:
Same with above